test-suite-data.json

```diff
@@ -550,5 +550,113 @@
     "qualifiers": null,
     "subpath": null,
     "is_invalid": false
+  },
+  {
+    "description": "brew names may contain at signs",
+    "purl": "pkg:brew/postgres%4016",
+    "canonical_purl": "pkg:brew/postgres%4016",
+    "type": "brew",
+    "namespace": null,
+    "name": "postgres@16",
+    "version": null,
+    "qualifiers": null,
+    "subpath": null,
+    "is_invalid": false
+  },
+  {
+    "description": "brew may contain multiple at signs",
+    "purl": "pkg:brew/postgres@16@16.1",
+    "canonical_purl": "pkg:brew/postgres%4016@16.1",
+    "type": "brew",
+    "namespace": null,
+    "name": "postgres@16",
+    "version": "16.1",
+    "qualifiers": null,
+    "subpath": null,
+    "is_invalid": false
+  },
+  {
+    "description": "brew may specify a namespace that uses the Homebrew default tap lookup mechanism",
+    "purl": "pkg:brew/some-org/some-tap/sqlite@3.43.2?tap_url=https://git.example.com/some-org/some-tap.git",
+    "canonical_purl": "pkg:brew/some-org/some-tap/sqlite@3.43.2?tap_url=https://git.example.com/some-org/some-tap.git",
+    "type": "brew",
+    "namespace": "https://git.example.com/some-org/some-tap.git",
```

Comment:
**suggestion (blocking):** Per [this comment](https://github.com/package-url/purl-spec/pull/281#discussion_r1421046349), I got this case wrong.

```suggestion
    "namespace": "some-org/some-tap",
```

The namespace should always be what's in first part, and it's up to the tooling reading this and calling `brew` to pass the explicit `tap_url` qualifier as the git URL for cloning.